### ISSUE TITLE
Warpify subshell causes Docker Desktop to reset in a loop

### ISSUE THREAD

**Issue by Alex-EEE** [warpdotdev/Warp#5539]
### Dupe Check

- [X] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates

### Describe the bug

When I try to warpify bash running in Docker Desktop on mac, Docker desktop opens and closes in an endless loop until I force quit Warp

### To reproduce

docker run .... bash

say yes to "Warpify subshell"

### Operating system

MacOS

### Operating system and version

14.6.1

### Regression

Yes, this bug started recently or with an X Warp version

### Recent working Warp date

mid Nov

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this an issue only in Warp?

Yes, I confirmed that this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by acarl005** [comment]
Hi @Alex-EEE thanks for reporting. To help me reproduce this issue, can you tell me some details about the the image you're running? The more the better.

**Comment by Alex-EEE** [comment]
hey @acarl005 , thanks for the quick response!  I actually think I just figured out what the crux is.

It actually only happens if you have a warp terminal running docker compose, where you get that little menu of buttons like "w: enable watch" etc
<img width="507" alt="image" src="https://github.com/user-attachments/assets/9d186981-2efa-4d30-906f-4da8b85cc642">

I tried it as a tab and as separate windows, and it still happens.  Also, I'm running the same image as I'm running inside docker compose.  So I think what's happening is somehow a "v" keystroke is getting from the other warp terminal, into the docker compose terminal, over and over.  (you can see warp and docker constantly changing context as the main window)

So to summarize, I think the repro is
- Start a container from image foo with docker compose (such that you get that little v, o, w menu at the bottom)
- Start a container using docker run, using image foo
- Warpify subshell

**Comment by acarl005** [comment]
I think the info provided is adequate so re-opening it

**Comment by Alex-EEE** [comment]
thanks!  I haven't tried this with docker compose with the later versions, I'll try to remember to give it a shot soon!
